library/psa_crypto.c

```diff
@@ -20,6 +20,10 @@
 
 #include "common.h"
 
+#if defined(MBEDTLS_PSA_CRYPTO_CONFIG)
+#include "check_crypto_config.h"
```

Comment:
I don't think `check_crypto_config.h` should be included if `MBEDTLS_PSA_CRYPTO_C` is not defined. What it checks wouldn't matter in that case.